lib/cosmos/interfaces/serial_interface.rb

```diff
@@ -50,6 +50,7 @@ def initialize(write_port_name,
       @write_allowed = false unless @write_port_name
       @write_raw_allowed = false unless @write_port_name
       @read_allowed = false unless @read_port_name
+      @flow_control = :NONE
```

Comment:
No.  I don't want to change the interface signature for backwards compat reasons.